src/Containers/VendorsList/Profile.tsx

```diff
@@ -5,16 +5,18 @@ import { HeaderRow, HeaderRowProps } from '../HeaderRow';
 import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
 import { flex, media } from '../../Utils/Mixins';
 
-export interface IProfileProps extends MainInterface, ResponsiveInterface,React.HTMLAttributes<HTMLDivElement> {
+export interface IProfileProps extends MainInterface, ResponsiveInterface, React.HTMLAttributes<HTMLDivElement> {
     key: number;
-    name: string;
-    email: string;
+    name?: string;
+    email?: string;
     imageUrl?: string;
     headerRowProps?: HeaderRowProps;
-    profileProps?: IFeaturedProfileProps;
+    profileProps?: Omit<IFeaturedProfileProps, 'key, background'>;
```

Comment:
shouldn't this be 'key' , 'background'? if this works i am probs confusing the syntax and you are correct lol

Reply:
Oops yeah haha I did do it wrong thanks for the catch